packages/gcp/changelog.yml

```
@@ -1,4 +1,9 @@
 # newer versions go on top
+- version: "2.35.1"
```

```suggestion
- version: "2.36.0"
```
Bumping `minor` version instead of `patch` makes sense as it was done [here](https://github.com/elastic/integrations/pull/9508) as well